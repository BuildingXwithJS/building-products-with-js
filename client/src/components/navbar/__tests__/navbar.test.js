import Navbar from '../index';

test('# Navbar', () => {
  const wrapperRoot = shallow(<Navbar current="/" />);
  expect(wrapperRoot).toMatchSnapshot();

  const wrapperCreate = shallow(<Navbar current="/create" />);
  expect(wrapperCreate).toMatchSnapshot();

  const wrapperUser = shallow(<Navbar user={{login: 'test'}} current="/profile/me" />);
  expect(wrapperUser).toMatchSnapshot();
});
