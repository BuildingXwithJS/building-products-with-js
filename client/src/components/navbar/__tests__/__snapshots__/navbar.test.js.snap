exports[`test # Navbar 1`] = `
<nav
  className="navbar navbar-default">
  <div
    className="container-fluid">
    <div
      className="navbar-header">
      <Link
        className="navbar-brand"
        onlyActiveOnIndex={false}
        style={Object {}}
        to="/">
        Brand
      </Link>
    </div>
    <ul
      className="nav navbar-nav">
      <li>
        <a>
          <b>
            Browse questions
          </b>
        </a>
      </li>
      <li>
        <Link
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/create">
          Create new question
        </Link>
      </li>
    </ul>
  </div>
</nav>
`;

exports[`test # Navbar 2`] = `
<nav
  className="navbar navbar-default">
  <div
    className="container-fluid">
    <div
      className="navbar-header">
      <Link
        className="navbar-brand"
        onlyActiveOnIndex={false}
        style={Object {}}
        to="/">
        Brand
      </Link>
    </div>
    <ul
      className="nav navbar-nav">
      <li>
        <Link
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/">
          Browse questions
        </Link>
      </li>
      <li>
        <a>
          <b>
            Create new question
          </b>
        </a>
      </li>
    </ul>
  </div>
</nav>
`;

exports[`test # Navbar 3`] = `
<nav
  className="navbar navbar-default">
  <div
    className="container-fluid">
    <div
      className="navbar-header">
      <Link
        className="navbar-brand"
        onlyActiveOnIndex={false}
        style={Object {}}
        to="/">
        Brand
      </Link>
    </div>
    <ul
      className="nav navbar-nav">
      <li>
        <Link
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/">
          Browse questions
        </Link>
      </li>
      <li>
        <Link
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/create">
          Create new question
        </Link>
      </li>
    </ul>
    <ul
      className="nav navbar-nav navbar-right">
      <li>
        <a>
          <b>
            Logged in as \"test\"
          </b>
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
